frontend/src/core/user/templates/userProfile.hbs

```diff
@@ -1,56 +1,58 @@
-                        <div class="inner display-none">
-                            <label for="password">New password
-                                <span class="req">*</span>
-                                <span id="passwordError" class="error feedback"></span>
-                            </label>
-
-                            <input type="password" class="width-15" id="password" value="" autocomplete="off">
-                            <input type="text" class="width-15 display-none" id="passwordText" value="" autocomplete="off">
-                            <button href="#" class="toggle-password display-none"><i class="fa fa-eye"></i></button>
-                        </div>
-                        <a href="#" class="btn primary change-password">{{t 'app.changepassword'}}</a>
-                    </div>
-                </div>
-                {{/ifValueEquals}}
+              <div class="form-group">
+                  <label>{{t 'app.email'}}
+                      <span id="titleErrorMessage" class="error"></span>
+                  </label>
+                  {{email}}
+              </div>
+
+              <div class="form-group">
+                  <label for="firstName">{{t 'app.firstname'}}
+                      <span class="req">*</span>
+                      <span id="firstNameError" class="error"></span>
+                  </label>
+                  <input type="text" class="width-30" id="firstName" value="{{firstName}}" autocomplete="off">
+              </div>
+
+              <div class="form-group">
+                  <label for="lastName">{{t 'app.lastname'}}
+                      <span class="req">*</span>
+                      <span id="lastNameError" class="error"></span>
+                  </label>
+                  <input type="text" class="width-30" id="lastName" value="{{lastName}}" autocomplete="off">
+              </div>
+
+              {{#ifValueEquals auth "local"}}
+              <div class="change-password-section">
+                  <div class="form-group">
+                      <div class="inner display-none">
+                          <label for="password">New password
+                              <span class="req">*</span>
+                              <span id="passwordError" class="error feedback"></span>
+                          </label>
+
+                          <input type="password" class="width-15" id="password" value="" autocomplete="off">
+                          <input type="text" class="width-15 display-none" id="passwordText" value="" autocomplete="off">
+                          <button href="#" class="toggle-password display-none"><i class="fa fa-eye"></i></button>
+                      </div>
+                      <a href="#" class="btn primary change-password">{{t 'app.changepassword'}}</a>
+                  </div>
+              </div>
+              {{/ifValueEquals}}
+
             </form>
         </div>
     </div>
```

Comment:
This looks really good to me! I think you are missing a closing </div> at the end of this file